### DISCUSSION TITLE
Linux Server Setup Extractors fail

### DISCUSSION THREAD

**Discussion by Binarycocaine** [TrinityCore/TrinityCore#28786] [Q&A] [answered]
hi guys,I'm following TrinityCore-wiki steps try to setup a linux server,and stucked here:

![image](https://user-images.githubusercontent.com/26806225/214593450-c6792fbc-b0e4-48c8-ba50-569e9dd2bcaa.png)

I'm using cloud server with Ubuntu 22.04.1

copy those files 

<
mapextractor
vmap4extractor
vmap4assembler
mmaps_generator
extractor.sh
>
 into wotlk3.3.5 client folder 

![image](https://user-images.githubusercontent.com/26806225/214595058-08bf319c-057c-496d-a763-e4037c62a392.png)

and try run command: sh extractor.sh , i got : 

![image](https://user-images.githubusercontent.com/26806225/214603827-b46f2f9b-8306-4201-bcda-f6d77631a63c.png)

I'm noob to bash commands.anyone can tell me what does it means  **Syntax error: "(" unexpected (expecting "}")**

**Comment by Jinnaix** [comment]
Im confused, you show a Screenshot with a wow client in folder structure 
/home/miller/wotlk3.3.5a_client/World of Warcraft

on the next screenshot, we see you executing and pointing to stuff in

/home/miller/client
/home/miller/server

**Comment by Aokromes** [comment]
don't use sh, use ./extractor.sh (change permisions to allow run) also don't use root to run tools.

**Comment by Shauren** [comment, marked as answer]
extractor.sh is supposed to be run by bash, not sh

**Comment by Binarycocaine** [comment]
thanks very much for bash command , it's now solved and extracting files.but got a new error:

![image](https://user-images.githubusercontent.com/26806225/214850023-7a72c2f3-b7f8-43ee-96da-c584d3198cfc.png)

**Reply by Binarycocaine** [reply]
fixed with created-user-auth
